satellite/console/service.go

```diff
@@ -241,6 +257,49 @@ func (s *Service) AddNewPaymentMethod(ctx context.Context, paymentMethodToken st
 	return pp, nil
 }
 
+func (s *Service) AttachPaymentMethodToProject(ctx context.Context, paymentMethodID []byte, projectID uuid.UUID) (err error) {
+	defer mon.Task()(&ctx)(&err)
+
+	authorization, err := GetAuth(ctx)
+	if err != nil {
+		return err
+	}
+
+	tx, err := s.store.BeginTx(ctx)
+	if err != nil {
+		return err
+	}
+
+	err = withTx(tx, func(tx DBTx) error {
+		projectPayment, err := tx.ProjectPayments().GetDefaultByProjectID(ctx, projectID)
+		if err != nil && err != sql.ErrNoRows {
+			return err
+		}
+
+		if projectPayment != nil {
```

Comment:
Is it possible for `projectPayment to be nil here`?

Reply:
yes.
first payment method creation.
no previous payment method.
err = sql.ErrNoRows
projectPayment = nil